next.config.js

```diff
@@ -61,5 +61,6 @@ module.exports = {
     NEXTAUTH_SECRET: process.env.NEXTAUTH_SECRET,
     NEXT_AUTH_USERNAME: process.env.NEXT_AUTH_USERNAME,
     NEXT_AUTH_PASSWORD: process.env.NEXT_AUTH_PASSWORD,
+    LOGGING_LEVEL: process.env.LOGGING_LEVEL,
```

Comment:
missing the change in the dockerfile and same on teamcity

Reply:
Added the LOGGING_LEVEL variable to the dockerfile and teamcity. Thanks!